apache-rat-tasks/src/site/apt/examples/basic.apt.vm

```diff
@@ -59,28 +58,47 @@ java -jar apache-rat-${project.version}.jar src
   The equivalent of
 
 +------------------------------------------+
-java -jar apache-rat-${project.version}.jar --addlicense
-  --copyright "Copyright 2008 Foo"
+java -jar apache-rat-${project.version}.jar --edit-license
+  --edit-copyright "Copyright 2008 Foo"
   /path/to/project
 +------------------------------------------+
 
   is
 
 ---
-<rat:report addLicenseHeaders="true"
-            coprightMessage="Copyright 2008 Foo">
+<rat:report editLicense="true"
+            editCopyright="Copyright 2008 Foo">
     <fileset dir="/path/to/project"/>
 </rat:report>
 ---
 
-* Create a styled HTML report
+* Create a styled report
+
+ There are two types of styled reports, both are activated by using the "outputStyled" attribute on the "rat:report" element.
+
+ The first is the standard report type.  This uses one of the standard reports shipped with Rat which are activated by setting the value of "outputStyled" to the name of the report.
+
+         *  plain-rat: The default style
+
+         * missing-headers: Produces a report of files that are missing headers
+
+         * unapproved-licenses: Produces a report of the files with unapproved licenses
+
+         * xml: The default style.
```

Comment:
As the default output is text I assume "xml" should have a different text or is the Ant version a combination of XML+RAT default report?